src/stories/HdRange.stories.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable import/no-extraneous-dependencies, no-console */
+import { storiesOf } from '@storybook/vue';
+
+import HdRange from 'hd-blocks/components/form/HdRange.vue';
+import FormWrapper from 'hd-blocks/storiesWrappers/FormWrapper';
+import { number } from '@storybook/addon-knobs';
+
+storiesOf('Form/HdRange', module)
+  .addDecorator(FormWrapper)
+  .add('default 🎛', () => ({
+    components: { HdRange },
+    template: `
+      <div>
+        <HdRange
+          :minValue=minValue
+          :maxValue=maxValue
+          :rangeStep=rangeStep
+          :value="value"
```

Comment:
To keep the same api for the form components (necessary for HdDynamicForm), we prefer to use `v-model`.
```suggestion
          v-model="value"
```

Reply:
I did not found a way to do it. I have to validate that value, so I can not use v-modal in the story, otherwise I get this 
![image](https://user-images.githubusercontent.com/19528860/67214441-0fa34580-f420-11e9-8293-3542ea2a59c1.png)

Reply:
I'm using the _v-model_ on the input element. The component still have _value_ as a prop.